pkgs/by-name/li/libtapi/package.nix

```diff
@@ -0,0 +1,178 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  ninja,
+  python3,
+  zlib,
+}:
+
+let
+  # Apple’s LLVM fork uses its own versioning scheme.
+  # See: https://en.wikipedia.org/wiki/Xcode#Toolchain_versions
+  # Note: Can’t use a sparse checkout because the Darwin stdenv bootstrap can’t depend on fetchgit.
+  appleLlvm = {
+    version = "15.0.0"; # As reported by upstream’s `tapi --version`.
+    rev = "2b5ff47e44b059c03de5779479d01a133ab4d581"; # Per the TAPI repo.
+    hash = "sha256-X37zBbpSEWmqtdTXsd1t++gp+0ggA8YtB73fGKNaiR0=";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libtapi";
+  version = "1500.0.12.3";
+
+  outputs = [
+    "out"
+    "bin"
+    "dev"
+  ];
+
+  srcs = [
+    (fetchFromGitHub {
+      name = "tapi-src";
+      owner = "apple-oss-distributions";
+      repo = "tapi";
+      rev = "tapi-${finalAttrs.version}";
+      hash = "sha256-YeaA2OeSY1fXYJHPJJ0TrVC1brspSvutBtPMPGX6Y1o=";
+    })
+    # Don’t pull down the entire repo when only a few projects are needed.
```

Comment:
this doesn't pull down the whole repo?

Reply:
You’re right. That’s an obsolete comment. I’ll update it to match the other ones about not being able to use `fetchgit` in the bootstrap.